Makefile

```
@@ -402,7 +449,7 @@ fwd-autotune:
 kubescape-vendor: 
```

checkmake (0.3.2)

[warning] Target body for "kubescape-vendor" exceeds allowed length of 5 lines (6).

(maxbodylength)